rust/platform/triple_mappings.bzl

```diff
@@ -140,6 +147,17 @@ def abi_to_constraints(abi):
     return []
 
 def triple_to_system(triple):
+    """Returns a system name for a given platform triple
+
+    Args:
+        triple (str): A platform triple. eg: `x86_64-unknown-linux-gnu`
+
+    Returns:
+        str: A system name
+    """
+    if triple == "wasm32-wasi":
+        return "wasi"
+
```

Comment:
With the name change from wasm32-unknown-wasi to wasm32-wasi, was there a broader change in how targets are specified, or is this still just kind of a snowflake?

What I'm trying to get at is whether or not this function, per-spec could just return the last section of the target or not.

Reply:
It looks that there is also a tuple for `{aarch64,x86_64}-fuchsia`, but I don't know whether that's a broader change or not.

https://doc.rust-lang.org/nightly/rustc/platform-support.html